src/libs/interfaces/generator/cpp_generator.cpp

```diff
@@ -1051,13 +1042,9 @@ CppInterfaceGenerator::write_methods_cpp(FILE *                      f,
 			        " * @param index index for of the value\n"
 			        " */\n"
 			        "void\n"
-			        "%s%s::set_%s(unsigned int index, const %s new_%s)\n"
+			        "%s%s::set_%s(unsigned int index, const %s new_%s)\n" // 9
```

Comment:
What's that `// 9` doing here?

Reply:
oh, that was some temporary comment to help me keep track of the argument indices. Can be removed.